<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qf::qmlwidgets::internal::PrintTableViewWidget</class>
 <widget class="QWidget" name="qf::qmlwidgets::internal::PrintTableViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="qf::qmlwidgets::SaveOptionsWidget" name="saveSettingsWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>template</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>file</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="edReportFileName"/>
            </item>
            <item>
             <widget class="QPushButton" name="btChooseReportFileName">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>resources</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="lstQrcReports"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>report</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>title</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="edReportTitle">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>note</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QTextEdit" name="edNote"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="grpColumns">
       <property name="title">
        <string>data</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="chkSelectedRowsOnly">
          <property name="text">
           <string>only selected rows</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qf::qmlwidgets::internal::TableViewChooseColumnsWidget" name="columnsChooser" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qf::qmlwidgets::internal::TableViewChooseColumnsWidget</class>
   <extends>QWidget</extends>
   <header location="global">src/internal/tableviewchoosecolumnswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qf::qmlwidgets::SaveOptionsWidget</class>
   <extends>QWidget</extends>
   <header location="global">src/saveoptionswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lstQrcReports</tabstop>
  <tabstop>edReportFileName</tabstop>
  <tabstop>btChooseReportFileName</tabstop>
  <tabstop>edReportTitle</tabstop>
  <tabstop>edNote</tabstop>
  <tabstop>chkSelectedRowsOnly</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
